com.obsproject.Studio.Plugin.VkCapture.metainfo.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.VkCapture</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>OBS Vulkan/OpenGL Capture</name>
```

Comment:
Since we named the corresponding one as `OBS VkCapture tools`, I think we can name the plugin itself `OBS VkCapture`